react/week1/my-app/src/components/Main.js

```diff
@@ -0,0 +1,12 @@
+import TodoList from './userItemExpanded'
+
+function Main(props) {
+    return (
+        <div>
+            <TodoList list={props.todo} />
+        </div>
+
+    )
+}
+
+export default Main
```

Comment:
I don't see this component present in your component tree (you don't import it anywhere). I guess then it is redundant.